openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsProvider/SuggestionsProvider.tsx

```diff
@@ -70,6 +70,7 @@ const SuggestionsProvider = ({ children }: { children?: ReactNode }) => {
     try {
       const { data } = await getSuggestionsList({
         entityFQN,
+        limit: 500,
```

Comment:
will this impact the performance? I guess we may have count from backend instead passing higher limit. cc: @pmbrull @karanh37